src/elements/table/Table.js

```diff
@@ -259,8 +259,8 @@ export class NovoTable {
      * @param filter
      */
     onFilterClick(column, filter) {
-        if (Array.isArray(column.filter)) {
-            if (column.filter.includes(filter)) {
+        if (Array.isArray(column.filter) && column.multiple) {
+            if (column.filter.indexOf(filter) > -1) {
```

Comment:
Can you tilde this instead? I think it's a preference:

`if (column.filter.indexOf(filter) > -1) {`

`if (~column.filter.indexOf(filter)) {`

[Fiddle](https://jsfiddle.net/krsween/kf7q5cjL/)